demos/node-classification/rgcn/rgcn-aifb-node-classification-example.ipynb

```diff
@@ -312,8 +318,8 @@
      "text": [
       "\n",
       "Test Set Metrics:\n",
-      "\tloss: 1.3316\n",
-      "\tacc: 0.5556\n"
+      "\tloss: 0.4581\n",
+      "\tacc: 0.8889\n"
```

Comment:
Seems to be a result random train/test split - with the previous run to this one (same code) I got 94.8% accuracy. The paper only uses one train/test split

Reply:
I take that back! increasing the number of parameters yields consistently better test results 94% +

Reply:
Ah, so you think that this is within the natural variance of the algorithm due to weight initialisation (since there's no other randomness in this algorithm, right?)?

> The paper only uses one train/test split

Are you referring to the train/test split from [1]? (That is, http://data.dws.informatik.uni-mannheim.de/rmlod/LOD_ML_Datasets/data/old/datasets/AIFB/) It looks like we're not using the same split?

Also, the paper says:

- "averaged over 10 runs": do we get similar accuracy averaged over 10 runs?
- "We further set aside 20% of the training set as a validation set for hyperparameter tuning": it seems like this demo still uses 80% for the training set?

[1] Ristoski, P.; de Vries, G. K. D.; and Paulheim, H. 2016. A collection of benchmark datasets for systematic evaluations of machine learning on the semantic web. In International Semantic Web Conference, 186–194. Springer

Reply:
With the initial parameters I had the natural weight + train test variance affected the final results significantly. It looks like with a larger number of parameters (layer sizes of 32 and 20 bases) the variance of final results decreases and we see results consistently comparable to the papers results.

And yeah, we're using a random split instead of their split, mostly because the single file dataset I downloaded was easier to handle than the one linked (but doesn't contain the train/test/val split).